.github/workflows/integration-tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         runs-on: [ubuntu-latest]
       fail-fast: false
 
-    runs-on: ${{ matrix.runs-on }}
+    runs-on: ${{ matrix.os }}
```

Comment:
The linter highlighted this as an error and suggested this change, have I broken anything?

Reply:
No idea